android/Package.swift

```diff
@@ -0,0 +1,2 @@
+import PackageDescription
+let package = Package()
```

Comment:
@mikasdo mentioned this line may not be necessary, is it?

Reply:
Removed. 🔥